extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -799,13 +864,17 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 				targetRow.push({
 					value: this._blobContainerDropdowns[index]
 				});
+				targetRow.push({
+					value: this._blobContainerLastBackupFileDropdowns[index]
+				});
 				data.push(targetRow);
 			});
 			this._blobContainerTargetDatabaseNamesTable.dataValues = data;
```

Comment:
To improve reliability, I suggest changing this to:
await this._blobContainerTargetDatabaseNamesTable.setDataValues(data);

Reply:
updated 
from
`this._blobContainerTargetDatabaseNamesTable.dataValues = data;`
to
`await this._blobContainerTargetDatabaseNamesTable.setDataValues(data);`